mathics/core/pattern.py

```diff
@@ -1,6 +1,11 @@
 # cython: language_level=3
```

Comment:
BTW - I think we should remove cython. At this point it is probably not speeding things up and getting in the way.

Reply:
For this, Cython was useful in detecting some errors when I renamed some arguments. Maybe we can remove this later.

Reply:
Ok - leave in then.